src/ui/components/Timeline/LoadingProgressBars.tsx

```diff
@@ -0,0 +1,43 @@
+import { clamp } from "lodash";
+import { useSelector } from "react-redux";
+import { getLoadedRegions } from "ui/reducers/app";
+import { getZoomRegion } from "ui/reducers/timeline";
+import { overlap } from "ui/utils/timeline";
+
+import styles from "./LoadingProgressBars.module.css";
+
+export default function LoadingProgressBars() {
+  const loadedRegions = useSelector(getLoadedRegions);
+  const zoomRegion = useSelector(getZoomRegion);
+
+  const endTime = zoomRegion ? zoomRegion.endTime : 0;
```

Comment:
I'm guessing that the false branch here is something we don't expect to hit? `0` seems like it would make for an odd value of `endTime` in the calculations below.

Reply:
`zoomRegion` can be null, so we need to account for it somehow. Agreed it wouldn't make much sense though. Probably best to just early-return `null` in this case.